docs/getting-started.md

```diff
@@ -5,7 +5,7 @@ slug: '/'
 
 # Getting Started
 
-[IaSQL](https://iasql.com) is an open-source SaaS to manage cloud infrastructure using a unmodified PostgreSQL database by maintaining a 2-way connection between the hosted database and your AWS account. The rows in the database tables represent the infrastructure in your cloud account. Our [CLI](/install) provisions a PostgreSQL database for you and configures it to manage an AWS account and region. The database is automatically backfilled with all your existing cloud resources. Which tables are loaded into an [IaSQL database](/database) is configured based on what [IaSQL modules](/module) are installed in a db. Every IaSQL module represents a cloud service like `aws_ec2` or `aws_elb`. Once the desired modules are installed, run `INSERT` or `UPDATE` queries on the database by using the PG connection string displayed when `iasql connect` completes with your preferred [PostgreSQL client](/connect). Finally run the `apply` CLI command to provision infrastructure in your cloud account based on the IaSQL db.
+[IaSQL](https://iasql.com) is an open-source SaaS to manage cloud infrastructure using a unmodified PostgreSQL database by maintaining a 2-way connection between the hosted database and your AWS account. The rows in the database tables represent the infrastructure in your cloud account. Our [dashboard](https://app.iasql.com) provisions a PostgreSQL database for you and configures it to manage an AWS account and region. The database is automatically backfilled with all your existing cloud resources. Which tables are loaded into an [IaSQL database](/database) is configured based on what [IaSQL modules](/module) are installed in a db. Every IaSQL module represents a cloud service like `aws_ec2` or `aws_elb`. Once the desired modules are installed with the `install` [IaSQL PostgreSQL function](/function), run `INSERT` or `UPDATE` queries on the database by using the PG connection string displayed when `iasql connect` completes with your preferred [PostgreSQL client](/connect). Finally run the `iasql_apply` PostgreSQL function to provision infrastructure in your cloud account based on the IaSQL db.
```

Comment:
```suggestion
[IaSQL](https://iasql.com) is an open-source SaaS to manage cloud infrastructure using a unmodified PostgreSQL database by maintaining a 2-way connection between the hosted database and your AWS account. The rows in the database tables represent the infrastructure in your cloud account. Our [dashboard](https://app.iasql.com) provisions a PostgreSQL database for you and configures it to manage an AWS account and region. The database is automatically backfilled with all your existing cloud resources. Which tables are loaded into an [IaSQL database](/database) is configured based on what [IaSQL modules](/module) are installed in a db. Every IaSQL module represents a cloud service like `aws_ec2` or `aws_elb`. Once the desired modules are installed with the `install` [IaSQL PostgreSQL function](/function), run `INSERT` or `UPDATE` queries on the database by using the PG connection string displayed when you first set up in the dashboard with your preferred [PostgreSQL client](/connect). Finally run the `iasql_apply` PostgreSQL function to provision infrastructure in your cloud account based on the IaSQL db.
```

Reply:
(I don't know why Github isn't diffing this line correctly. There was one last `cli` reference in there that I replaced with a reference to the dashboard)

Reply:
oof, I see it now. Good catch!